recipes/r-wmtsa/meta.yaml

```diff
@@ -19,6 +19,7 @@ build:
   rpaths:
     - lib/R/lib/
     - lib/
+  skip: True  # [win]
```

Comment:
Is win32 enough to skip?

Reply:
No, as far as I remember the build failed on win64 too.

Reply:
Would be nice if you could add a comment with he win32 error, so people interested on win32 builds could fix it.

Reply:
win64 you mean? I'll see what the error was and add a comment.

Reply:
Could you commit this one as is? I'll need to re-run the Travis build to check why win64 fails, I think that's easier from within a feedstock.